src/App.js

```diff
@@ -33,6 +34,7 @@ const App = () => {
           <Projects/>
       </Route>
       <Route exact path="/Data" component={Data} />
+      <Route exact path="/heatmap" component={Heatmap} />
```

Comment:
I think the 'Heatmap' should be a component in the /components directory to be rendered onto the data page, and not a page itself. This is because we will be rendering the Heatmap within the data page.